index.html

```diff
@@ -41,6 +41,43 @@ <h2 class="about-header">about-me</h2>
 			</div>
 		</main>
 		<section id="projects" tabindex="2">
-			
+			<div class="projects-gallery">
```

Comment:
Indentation is v deep, in code above (that I can't see) is everything as set back as poss? Just in case you can move main back so its not so annoying for you to read (I assume its been indented correctly but just in case)